src/bio2bel/sources/intact.py

```diff
@@ -350,6 +341,25 @@ def _add_my_row(graph: BELGraph, row) -> None:
                         identifier='0018256', ),
                 )
 
+            # demethylation reaction
+            elif relation == 'demethylation reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification('Me'),
+                )
+
+            # deneddylation reaction
+            elif relation == 'deneddylation reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification('Nedd'),
+                )
+
+            # protein modification
+            elif relation in PROTEIN_MOD_DICT.keys():
+                abbreviation = PROTEIN_MOD_DICT[relation]
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(abbreviation),
+                )
+
```

Comment:
there should be an `else` at the bottom that warns there's something unhandled.

Reply:
@sophiakrix best practice is mark the comment as resolved with the commit, then close

Closed with d4e56483f4319e6c226b7a826419096a66e34bff